front-end/src/utilities/EventUtilities.jsx

```diff
@@ -134,5 +134,14 @@ export const timeZoneAbbreviations = [
     "Pacific/Honolulu",
     "Pacific/Niue",
   ];
+
+
+
+  //view volunteer roles for specific event
+export const volunteerRoles = async (eventID) => {
+  const response = await api.get(`events/${eventID}/volunteers/`);
+  let rolesArr = response.data;
+  return rolesArr;
+};
   
```

Comment:
this call is actually meant for the admin page to give all of the role data the event details will have all of the role data that you need for the volunteer application.  This is great to have here for the admin page though!

Reply:
i'm going to keep it like that for now until that's completed so we can see the roles for now. you can create an issue to update it when that's done.